examples/run_CPMG.py

```diff
@@ -10,116 +10,131 @@
 from pyspecdata import *
 from numpy import *
 import SpinCore_pp
-from SpinCore_pp.ppg import run_cpmg
+from SpinCore_pp import prog_plen
+from SpinCore_pp.ppg import generic
 import os
 from datetime import datetime
 import h5py
-raise RuntimeError("This pulse proram has not been updated.  Before running again, it should be possible to replace a lot of the code below with a call to the function provided by the 'generic' pulse program inside the ppg directory!")
+from Instruments.XEPR_eth import xepr
 
-fl = figlist_var()
 # {{{importing acquisition parameters
 config_dict = SpinCore_pp.configuration("active.ini")
-nPoints = int(config_dict["acq_time_ms"] * config_dict["SW_kHz"] + 0.5)
+config_dict["SW_kHz"] = 75e6/round(75e6/config_dict["SW_kHz"]/1e3)/1e3
+nPoints = int(config_dict["echo_acq_ms"] * config_dict["SW_kHz"] + 0.5)  # per echo
+target_directory = getDATADIR(exp_type="ODNP_NMR_comp/Echoes")
+config_dict["echo_acq_ms"] = nPoints / config_dict["SW_kHz"]
 # }}}
 # {{{create filename and save to config file
 date = datetime.now().strftime("%y%m%d")
 config_dict["type"] = "CPMG"
 config_dict["date"] = date
 config_dict["cpmg_counter"] += 1
-filename = f"{config_dict['date']}_{config_dict['chemical']}_{config_dict['type']}"
+filename = (
+    f"{config_dict['date']}_{config_dict['chemical']}_generic_{config_dict['type']}"
+)
+# }}}
+# {{{let computer set field
+print(
+    "I'm assuming that you've tuned your probe to",
+    config_dict["carrierFreq_MHz"],
+    "since that's what's in your .ini file",
+)
+Field = config_dict["carrierFreq_MHz"] / config_dict["gamma_eff_MHz_G"]
```

Comment:
can we not capitalize this? it's inconsistent with how we do all our other variables.